homeassistant/components/alexa/entities.py

```diff
@@ -853,6 +854,7 @@ def interfaces(self):
 
 
 @ENTITY_ADAPTERS.register(input_number.DOMAIN)
+@ENTITY_ADAPTERS.register(number.DOMAIN)
 class InputNumberCapabilities(AlexaEntity):
     """Class to represent input_number capabilities."""
 
```

Comment:
On line 869 it references `input_number.DOMAIN`, should that be adjusted to also be number?

Reply:
It should use `self.entity.domain` there.

Reply:
I guess my thought process was from Alexa's perspective there's really no difference between an input number and a number. So leaving the instance id like this allowed numbers to report all the same capabilities and fields.

But I suppose it may no longer be unique like this. If `input_number.a` and `number.a` both exist this will break. I'll change it and add new conditionals to the other places that check specifically for the input number domain.

Reply:
May be you can add tests as-well ?